bootstrap.sh

```diff
@@ -23,24 +24,33 @@ last_bootstrapping_line_number=$(grep --line-number '^## Writing the Legislation
 
 cd ..
 pwd
-mv country-template openfisca-$lowercase_country_name
+mv country-template-master openfisca-$lowercase_country_name
 cd openfisca-$lowercase_country_name
 
 git init
 git add .
 git commit --message 'Initial import from OpenFisca country-template' --author='OpenFisca Bot <bot@openfisca.org>'
 
-sed -i '' "3,${last_bootstrapping_line_number}d" README.md  # remove instructions lines
-sed -i '' "s|country_template|$lowercase_country_name|g" README.md setup.py check-version-bump.sh Makefile `find openfisca_country_template -type f`
-sed -i '' "s|country-template|$lowercase_country_name|g" README.md
-sed -i '' "s|Country-Template|$COUNTRY_NAME|g" README.md setup.py check-version-bump.sh .github/PULL_REQUEST_TEMPLATE.md CONTRIBUTING.md `find openfisca_country_template -type f`
-sed -i '' "s|https://github.com/openfisca/openfisca-country-template|$URL|g" setup.py
+all_module_files=`find openfisca_country_template -type f`
+
+set -x
+
+# Use intermediate backup files due to lack of portable 'no backup' option. See https://stackoverflow.com/q/5694228/594053
+sed -i '.template' "s|country_template|$lowercase_country_name|g" README.md setup.py .circleci/config.yml Makefile $all_module_files
+sed -i '.template' "s|Country-Template|$COUNTRY_NAME|g" README.md setup.py .github/PULL_REQUEST_TEMPLATE.md CONTRIBUTING.md $all_module_files
+sed -i '.template' -e "3,${last_bootstrapping_line_number}d" README.md  # remove instructions lines
```

Comment:
This doesn't seem to work in `BSD/Sed`, I'm getting this error:

```
++ sed -i .template -e 3,d README.md
sed: 1: "3,d
": expected context address
```

Reply:
Mmh the problem doesn't come from BSD `sed` but from an empty `last_bootstrapping_line_number` 🤔

Could you please copy the log from the `grep` that computes that variable?